### ISSUE TITLE
class heritage EventEmitter is not an object or null

### ISSUE THREAD

**Issue by florian** [mozilla/shield-studies-addon-utils#228]
When I call `browser.study.setup`, I get the following error:
```
class heritage EventEmitter is not an object or null api.js:8222
Error: An unexpected error occurred main.js:3:27 
```

The exact same code worked a week ago, so it looks like a recent change in Firefox broke this.
@rhelmer investigated a bit and found that `EventEmitter` was moved from `ExtensionUtils` to `ExtensionCommon`.

```javascript
ChromeUtils.import("resource://gre/modules/ExtensionUtils.jsm");
const { EventEmitter, ExtensionError } = ExtensionUtils;
EventEmitter // => undefined
```

The same code with ExtensionCommon works as expected.
I tried to create a PR for this, but `npm run build` errors out on the current master version of this repo, so it's hard to test the change.

**Comment by gregglind** [comment]
Hm, thank you will fix.

Can you explain the FX version logic here?

Are you proposing a double try block to handle backwards compat?

PR welcome even if you can’t prove it works.

**Comment by motin** [comment]
This was fixed in 5.0.3
